src/components/InlineActionsContainer.vue

```diff
@@ -34,17 +34,45 @@ import { defineComponent } from 'vue'
  */
 export default defineComponent({
 	name: 'InlineActionsContainer',
+	data() {
+		return {
+			observer: new MutationObserver((mutations) => this.onMutation(mutations[0].target)),
+		}
+	},
+	mounted() {
+		const target = this.$el.getElementsByClassName('inline-container-content')[0]
+		this.observer.observe(target, { childList: true })
+		this.onMutation(target)
+	},
+	beforeDestroy() {
+		this.observer.disconnect()
+	},
+	methods: {
+		/**
+		 * Handle new or removed child nodes, recalculate the width
+		 *
+		 * @param {Node} target The mutation target
+		 */
+		onMutation(target) {
+			const children = [...target.childNodes].filter(node => node instanceof HTMLElement)
+			children.forEach(node => {
+				node.style.width = `${Math.round(100 / children.length)}%`
+			})
+		},
+	},
```

Comment:
This seems overly complicated?
Let me suggest a css only fix

Reply:
https://github.com/nextcloud/text/pull/3544#discussion_r1044635494

Reply:
Thank you!